### PR TITLE
v5.36 Notice

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: notices.json [hunk 1]
```diff
@@ -37,21 +37,21 @@
...
+      "serverVersion": ["<5.36"],
       "instanceType": "onprem",
-      "displayDate": ">= 2021-05-21T00:00:00Z"
+      "displayDate": ">= 2021-06-21T00:00:00Z"
     },
     "localizedMessages": {
       "en": {
         "title": "New server version available",
-        "description": "The new feature release offers shared channels, stakeholder overview and ad hoc tasks for incident collaboration, file search, and more. [Please also see the important upgrade notes](https://docs.mattermost.com/administration/important-upgrade-notes.html).",
+        "description": "The new feature release offers updates to Incident Collaboration with enhanced permission controls and automated welcome messages, and the addition of Hungarian language support. Focalboard project management is now integrated with Mattermost for the first time as a beta feature. [Upgrading](https://docs.mattermost.com/administration/upgrade.html) only takes a few minutes.",
```
Comment (esethna) [inline notices.json:51]: @amyblais I would propose adding @itao and @chenilim to review this, given this release is mainly about their features.

**Review by esethna — DISMISSED** [review]
Notice conditions look good, will leave it to the other PMs to comment on copy

**Comment by chenilim** [comment]
This message looks good @amyblais, except that if the Admin wants to enable Focalboard, they may struggle to find the steps.

@justinegeffen, I'm 2/5, but what do you think about updating the [Mattermost upgrade instructions](https://docs.mattermost.com/administration/upgrade.html) with a note on [how to enable Focalboard Beta](https://www.focalboard.com/download/mattermost/)?

**Review by chenilim — DISMISSED** [review]

**Comment by amyblais** [comment]
> This message looks good @amyblais, except that if the Admin wants to enable Focalboard, they may struggle to find the steps.
> 
> @justinegeffen, I'm 2/5, but what do you think about updating the [Mattermost upgrade instructions](https://docs.mattermost.com/administration/upgrade.html) with a note on [how to enable Focalboard Beta](https://www.focalboard.com/download/mattermost/)?

In the meanwhile I've linked to https://docs.mattermost.com/administration/important-upgrade-notes.html which also has the enabling instructions (among other places).

**Comment by amyblais** [comment]
@itao @ogi-m Kind reminder for reviews.

**Review by itao — APPROVED** [review]

**Review by ogi-m — APPROVED** [review]

**Pull request merged**